test/unit/sayt.ts

```diff
@@ -145,4 +145,24 @@ suite('Sayt', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveAlias })
       sayt.setInactive();
     });
   });
+
+  describe('checkRootNode()', () => {
+    it('should not setInactive if target found', () => {
```

Comment:
we like using parentheses to denote methods
`'should not call setInactive() if target found'`